src/javascript/azriakmalk-SquareArea.js

```diff
@@ -0,0 +1,4 @@
+//the function is count square area
+const Square = (n) => {
```

Comment:
@azriakmalk rename the function to be as same as in the filename, or vice versa.

Reply:
@JazibJafri okey